examples/pygi_mpl_numpy/README.rst

```diff
@@ -66,3 +66,19 @@ The include section requires the Python packages numpy, matplotlib (which are ta
         dateutil
         pyparsing
 
+Compiling
```

Comment:
You can 'compile' all sorts of things - `glib-compile-schemas` is actually closer to the original sense of bringing together separate pieces to make one unified thing. But in computing, it's come to also mean converting a high-level representation like source code into a lower level one like machine code or bytecode (the c in .pyc files stands for compiled - they contain Python bytecode). And most commonly it means turning C source code into machine code, which requires big dependencies and lots of complexity, so Python programmers, myself included, mostly avoid it as much as possible.